src/Sulu/Bundle/AdminBundle/Admin/AdminPool.php

```diff
@@ -58,6 +58,11 @@ public function getSecurityContexts()
         return $contexts;
     }
 
+    /**
+     * Returns the combined security contexts with placeholders.
+     *
+     * @return array
```

Comment:
Would like to move this into a seperate pull request where we maybe target the `getSecurityContextsWithPlaceholder()` and `getSecurityContexts` correctly on the `Admin` and `AdminPool` class.

The structure is nearly always the same for getSecurityContexts

System (string) -> Section (string) -> Context (string) -> Permissions (string).

Which ends in:

```php
array<string, array<string, array<string, string[]>>>
```

Reply:
Wouldn't it maybe also make sense to refactor out an interface as the `Admin` and the `AdminPool` are using the same methods.

Reply:
I think we should split this merge request up.

Reply:
Has been split up in #7181 and will be removed from here.